helpers/prism.js

```diff
@@ -255,19 +256,21 @@ export const codeStyles = css`
 		background-color: transparent;
 	}
 
-	pre[class*="language-"].line-numbers {
+	pre[class*="language-"].d2l-code.line-numbers {
 		counter-reset: linenumber;
 		padding-left: 3.8em;
 		position: relative;
 	}
 
-	pre[class*="language-"].line-numbers > code {
+	pre[class*="language-"].d2l-code.line-numbers > code {
 		position: relative;
 		white-space: inherit;
 	}
 
-	.line-numbers .line-numbers-rows {
-		border-right: 1px solid var(--d2l-code-line-numbers-separator);
+	.d2l-code.line-numbers > code > .line-numbers-rows {
+		background-color: var(--d2l-code-line-numbers-background);
```

Comment:
I didn't call it out in the PR description, but the design changed for the line-number formatting from:
![image](https://user-images.githubusercontent.com/9042472/177352233-6597920c-8731-4bff-8efe-369a2ab30a0a.png)
to:
![image](https://user-images.githubusercontent.com/9042472/177352301-5c0f008a-237c-47cc-bc36-f017ea72413f.png)